locales/en/kafka.json

```diff
@@ -4,10 +4,29 @@
   "clientID": "Client ID",
   "connection_rate": "Connection rate",
   "connection_rate_value": "up to 100 connections/second",
+  "connection_tab": {
+    "access_tab": "Access tab.",
+    "alter_allow": "The owner, the organization administrator, or users with the Allow Alter permission on this instance can manage access.",
+    "authentication_method": "Authentication method",
+    "bootstrap_server": "Bootstrap server",
+    "bootstrap_server_description": "Establish the initial connection between your client and Kafka instance.",
+    "create_service_account_to_generate_credentials": "Create a service account to generate credentials. Manage the service accounts you create on the <value>Service Accounts</value> page",
+    "current_instance": "Accounts do not have access to this instance unless granted under the instance’s",
+    "drawer_resource_tab_body_description_1": "Your application or tool initially connects to the Kafka instance using the bootstrap server and authenticates with your service accounts credentials.",
```

Comment:
The last part of this statement should be:
> with your service account credentials.

i.e. remove the "s" at the end of "account"